### PR TITLE
Tests-key-store: differentiate between account migration on iOS and n…

### PR DESCRIPTION
**Pull request by danimoh**
…on-iOS

### CODE REVIEW

File: tests/lib/KeyStore.spec.js [hunk 1]
```diff
@@ -111,20 +111,55 @@ describe('KeyStore', () => {
...
+        let migrationCookieDeleted = false,
+            accountsCookieDeleted = false;
+        spyOnProperty(document, 'cookie', 'set').and.callFake((/** @type {string} */ cookie) => {
+            if (cookie.startsWith('migrate=')) {
+                migrationCookieDeleted = true;
+            } else if (cookie.startsWith('accounts=')) {
+                accountsCookieDeleted = true;
+            }
+            expect(migrationCookieDeleted || accountsCookieDeleted).toBe(true);
+            expect(cookie.includes('expires=Thu, 01 Jan 1970 00:00:01 GMT')).toBe(true);
```
Comment (sisou) [inline tests/lib/KeyStore.spec.js]: I'd put this expect before the if-else condition.

Comment (danimoh) [reply tests/lib/KeyStore.spec.js]: What's the difference?

**Comment by danimoh** [comment]
Setting of the migration cookie is tested in the IframeApi specs and for the PopupApi (that triggers doMigrateAccountsToKeys) there are no tests yet.

File: tests/lib/KeyStore.spec.js (hunk 1 above)
Comment (sisou) [reply tests/lib/KeyStore.spec.js]: The difference is only a technicality: that you test first if the `set` is a deletion, before setting the `...deleted` variable.

Comment (svub) [reply tests/lib/KeyStore.spec.js]: it's also nice to have all expects together (code style / readability). No strong opinion though

**Pull request merged**